8_september/ts_assignment/Script/eleven.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+var _a;
```

Comment:
use proper variable name

Reply:
Updated variable names

Reply:
Not seeing any update. Still see variable _a. Use should write proper variable name instead of using like a , b ,c

Reply:
Applicable to all files